locksmith/src/operations/metadataOperations.ts

```diff
@@ -61,7 +61,7 @@ const getBaseTokenData = async (address: string) => {
   })
 
   let assetLocation = Asset.tokenMetadataDefaultImage({
-    base: baseURIFragement,
+    base: config.locksmithUri,
```

Comment:
I think this is still required, the use case this supports is around  a user providing a default image for their  token  (lock) being host in the organization provided storage